SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1283,11 +1288,20 @@ void FSpatialGDKEditorToolbarModule::OnBuildSuccess()
 		}
 		else
 		{
+			OnStartCloudDeploymentFinished();
 			OnShowFailedNotification(TEXT("Failed to launch cloud deployment. Unable to authenticate with SpatialOS."));
 		}
 	});
 }
 
+void FSpatialGDKEditorToolbarModule::OnStartCloudDeploymentFinished()
+{
+	AsyncTask(ENamedThreads::GameThread, [this]
```

Comment:
does this need to be in an async task?

Reply:
Yes, because the completion callback is called in a background thread.